aws/resource_aws_autoscaling_policy_test.go

```diff
@@ -375,3 +415,79 @@ resource "aws_autoscaling_policy" "foobar_simple" {
 }
 `, name, name, name)
 }
+
+func testAccAwsAutoscalingPolicyConfig_TargetTracking_Predefined(rName string) string {
+	return fmt.Sprintf(`
+resource "aws_launch_configuration" "test" {
+  name          = "tf-test-%s"
+  image_id      = "ami-21f78e11"
+  instance_type = "t1.micro"
```

Comment:
AFAIK `t1` family of instances has been deprecated - do you mind changing this to any low-price size of current family? e.g. `t2.micro`? https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/concepts_micro_instances.html